tests/test_aten_functions.py

```diff
@@ -2331,3 +2331,47 @@ def fn(x):
         match="The implementation of aten.max_pool2d_with_indices doesn't support returning indices yet.",
     ):
         check_functions_are_equivalent(fn, device, [x])
+
+
+# aten._log_softmax(Tensor self, int dim, bool half_to_float) -> Tensor
+@pytest.mark.parametrize("dtype", [torch.float32])
+@pytest.mark.parametrize("dim", [-1])
+def test_log_softmax_basic(device: str, dtype: torch.dtype, dim: int):
+    """Test _log_softmax basic functionality.
+
+    Note: Currently only testing CPU and limited CUDA configs due to MAX GPU reduction
```

Comment:
Since you're using the `device` fixture, this test runs on the gpu AND cpu, this comment is wrong